galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/GherkinTestRunner.java

```diff
@@ -457,18 +458,18 @@ private void markWaiting(@NotNull IFramework framework) throws TestRunException
         }
     }
 
-    private void updateStatus(String status, String timestamp) throws TestRunException {
+    private void updateStatus(TestRunLifecycleStatus status, String timestamp) throws TestRunException {
 
-        this.testStructure.setStatus(status);
-        if ("finished".equals(status)) {
+        this.testStructure.setStatus(status.toString());
+        if ("finished".equals(status.toString())) {
```

Comment:
Instead of asking "finished" whether it is equal to status.toString()... it's better to ask status whether it is equal to the "finished" string ? If we must...

Reply:
Should the comparison be ignoring the case of the strings ?

Reply:
I think so? I overrode the toString method to return the lowercase value of the enum if I'm understanding correctly. Do you think it would be best to explicitly say that status should be lowercase? or do you mean that .equals ignores case?